src/main/java/com/google/neighborgood/helper/TaskGroup.java

```diff
@@ -19,101 +19,65 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.EntityNotFoundException;
 import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
-import java.sql.Timestamp;
-import java.text.SimpleDateFormat;
+import com.google.neighborgood.User;
+import com.google.neighborgood.task.Task;
+import java.util.ArrayList;
 import java.util.HashMap;
 
 /**
  * Helper class that stores the HTML depiction of tasks in groups of 10 or less along with whether
  * or not the end of the query has been reached
  */
 public class TaskGroup {
-  private static HashMap<String, String>
-      usersNicknames; // Stores task owner's user info to prevent querying multiple times in
+  private static HashMap<String, User>
```

Comment:
nit: I would suggest declaring the type as Map instead of HashMap since we can change the underlying object to be a different kind of map without breaking any part of the code. If we declare it as HashMap and wrote some code that depends on the feature that is specific to HashMap, we may have to change the contract for TaskGroup class after changing the implementation.

Reply:
Sounds good, I will make these changes! :)